osu.Game.Tests/Visual/Gameplay/SkinnableHUDComponentTestScene.cs

```diff
@@ -19,7 +18,7 @@ public abstract class SkinnableHUDComponentTestScene : SkinnableTestScene
         {
             SetContents(skin =>
             {
-                var implementation = skin != null
+                var implementation = skin is not TrianglesSkin
```

Comment:
Isn't this meant to be `skin is LegacySkin`? Wouldn't imagine argon skin to use legacy implementation if/when we add it to skinnable test scene.
```suggestion
                var implementation = skin is LegacySkin
```

Reply:
It's really much of a muchness. I'll change it but it's going to require further changes in the future as triangles will not use the same `Default` hud elements.

Not a huge fan of the structure of this tests anyway.

Reply:
Yeah now that I read it again it doesn't matter at this point, at first glance I thought it was incorrect because it creates legacy components for non-legacy skin, but on the other hand it would create triangles components for non-triangles skin.

Disregard my comment.